cmd/ndm_daemonset/controller/controller.go

```diff
@@ -73,6 +73,8 @@ const (
 	NDMDeviceTypeKey = "ndm.io/blockdevice-type"
 	// NDMManagedKey specifies blockdevice cr should be managed by ndm or not.
 	NDMManagedKey = "ndm.io/managed"
+	// NDMZpoolName specifies the zpool name
+	NDMZpoolName = "ndm.io/zpool-name"
```

Comment:
Why is this change in this PR?

Reply:
Can the zpool related changes be moved to a separate PR.

Reply:
ok